framework/src/main/java/com/musala/sdcs/repository/ChannelRepository.java

```diff
@@ -33,14 +34,39 @@ public ChannelRepository() {
 	 * @param command
 	 * @param id
 	 */
-	public void updateChannelCommand(String command, Integer id) {
+	public String updateChannelCommand(String command, Integer id) {
 		String query = String.format(QUERY_UPDATE_COMMAND, command, id);
+		System.out.println(query);
 		try {
 			CallableStatement stmt = dbConnection.prepareCall(query);
 			stmt.executeUpdate();
+			return command;
 
 		} catch (SQLException e) {
 			logger.error(LOGGER_INVALID_SQL_MESSAGE, e);
+			System.out.println(query);
+			return "failed";
```

Comment:
Where do you expect to get this returned string "failed" and what actions do you perform when this "failed" is returned?